apps/eaas-api/src/app/middleware/authorizeInvitationSender.ts

```diff
@@ -2,6 +2,7 @@ import { NextFunction, Response } from "express";
 import { RequestWithUser } from "src/types/EaasUser";
 import { userService } from "../services/user.service";
 import { apiErrors } from "../utils/errors";
+// import { UserType } from "../../../../common/types"; TODO: this import is not working
```

Comment:
for now lets duplicate the ENUM on front-end / back-end -- can discuss strategies for sharing types between FE/BE